protoc-gen-openapiv2/defs.bzl

```diff
@@ -254,7 +259,16 @@ protoc_gen_openapiv2 = rule(
             mandatory = False,
             doc = "if set, the object's OpenAPI names will use the fully" +
                   " qualified names from the proto definition" +
-                  " (ie my.package.MyMessage.MyInnerMessage",
+                  " (ie my.package.MyMessage.MyInnerMessage)",
+        ),
+        "openapi_naming_strategy": attr.string(
+            default = "",
+            mandatory = False,
+            values = ["", "simple", "legacy", "fqn"],
+            doc = "configures how OpenAPI names are determined." +
+                  " Allowed values are `` (empty), `simple`, `legacy` and `fqn`." +
+                  " If unset, either `legacy` or `fqn` are selected, depending" +
+                  " on the value of the `fqn_for_openapi_name` setting",
```

Comment:
Could you copy the descriptions from the registry definitions to this doc too please? Can't hurt to copy things around a bit if it gives us a greater chance of a user reading it 😄.

Reply:
Can I use line breaks here? My fear is that inserting the descriptions inline without line breaks could really hurt readability.

Reply:
I frankly have no idea 😂. Looks like you can use newline literals: https://docs.bazel.build/versions/master/skylark/lib/string.html.